src/algokit_utils/beta/account_manager.py

```diff
@@ -0,0 +1,196 @@
+    #         never commit it into source control and ideally load it from the environment (ideally via a secret storage service) rather than the file system.
+    #     :param sender: The optional sender address to use this signer for (aka a rekeyed account)
+    #     :return: The account
+    #     """
+    #     account = mnemonic_account(mnemonic_secret)
+    #     return self.signer_account(rekeyed_account(account, sender) if sender else account)
+
+    def from_kmd(
+            self,
+            name: str,
+            predicate: Callable[[dict[str, Any]], bool] | None = None,
+    ) -> AddressAndSigner:
+        """
+        Tracks and returns an Algorand account with private key loaded from the given KMD wallet (identified by name).
+
+        Example (Get default funded account in a LocalNet):
+            default_dispenser_account = account.from_kmd('unencrypted-default-wallet',
+                lambda a: a['status'] != 'Offline' and a['amount'] > 1_000_000_000
+            )
+
+        :param name: The name of the wallet to retrieve an account from
+        :param predicate: An optional filter to use to find the account (otherwise it will return a random account from the wallet)
+        :return: The account
+        """
+        account = get_kmd_wallet_account(
+            name=name, predicate=predicate, client=self._client_manager.algod, kmd_client=self._client_manager.kmd
+        )
+        if not account:
+            raise ValueError(f"Unable to find KMD account {name}{' with predicate' if predicate else ''}")
+
+        self.set_signer(account.address, account.signer)
+        return AddressAndSigner(address=account.address, signer=account.signer)
+
+    # TODO
+    # def multisig(
+    #     self, multisig_params: algosdk.MultisigMetadata, signing_accounts: Union[algosdk.Account, SigningAccount]
+    # ) -> TransactionSignerAccount:
+    #     """
+    #     Tracks and returns an account that supports partial or full multisig signing.
+
+    #     Example:
+    #         account = account.multisig(
+    #             {
+    #                 "version": 1,
+    #                 "threshold": 1,
+    #                 "addrs": ["ADDRESS1...", "ADDRESS2..."]
+    #             },
+    #             account.from_environment('ACCOUNT1')
+    #         )
+
+    #     :param multisig_params: The parameters that define the multisig account
+    #     :param signing_accounts: The signers that are currently present
+    #     :return: A multisig account wrapper
+    #     """
+    #     return self.signer_account(multisig_account(multisig_params, signing_accounts))
+
+    def random(self) -> AddressAndSigner:
+        """
+        Tracks and returns a new, random Algorand account with secret key loaded.
+
+        Example:
+            account = account.random()
+
+        :return: The account
+        """
+        [sk, addr] = generate_account()
```

Comment:
NIT: Using tuple unpacking here feels more natural

Reply:
Done in de9b9d9